src/portable/synopsys/dwc2/dcd_dwc2.c

```diff
@@ -923,6 +923,19 @@ void dcd_edpt_clear_stall (uint8_t rhport, uint8_t ep_addr)
   }
 }
 
+void dcd_get_edpt_max_count(uint8_t rhport, uint8_t *in_edpt_max_count, uint8_t *out_edpt_max_count)
+{
+  uint8_t const max_ep_count = _dwc2_controller[rhport].ep_count;
```

Comment:
Just a small nitpick:
```suggestion
  const uint8_t max_ep_count = _dwc2_controller[rhport].ep_count;
```
Because we can have constant pointer to constant data and it will be like: 
`const uint8_t const * addr; `
So the place after a type is usually used for const modifier relates to the pointer.